charts/kube-prometheus-stack/values.yaml

```diff
@@ -648,6 +648,9 @@ alertmanager:
 ##
 grafana:
   enabled: true
+  # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
```

Comment:
i don´t think that info is needed as the ingress is disabled by default.
the comment is also missleading, as it should be under .Values.grafana.ingress.ingressClassName